CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contribution guidelines
 
 So you want to hack on Knative `pkg`? Yay! Please refer to Knative's overall
-[contribution guidelines](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md)
+[contribution guidelines](https://github.com/knative/community/blob/master/README.md)
```

Comment:
https://knative.dev/contributing/ looks like a proper link

Reply:
Thank you, fixed it!